wire-ios-data-model/Tests/OneOnOne/Mocks/MockActorOneOnOneMigrator.swift

```diff
@@ -0,0 +1,64 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+@testable import WireDataModel
+
+actor MockActorOneOnOneMigrator: OneOnOneMigratorInterface {
+
+    // MARK: - Life cycle
+
+    init() {}
+
+    // MARK: - migrateToMLS
+
+    var migrateToMLSUserIDIn_Invocations: [(userID: QualifiedID, context: NSManagedObjectContext)] = []
+    var migrateToMLSUserIDIn_MockError: Error?
+    var migrateToMLSUserIDIn_MockMethod: ((QualifiedID, NSManagedObjectContext) async throws -> MLSGroupID)?
+    var migrateToMLSUserIDIn_MockValue: MLSGroupID?
+
+    func setMigrateToMLSUserIDIn_MockError(_ error: Error?) {
+        migrateToMLSUserIDIn_MockError = error
+    }
+
+    func setMigrateToMLSUserIDIn_MockMethod(_ method: ((QualifiedID, NSManagedObjectContext) async throws -> MLSGroupID)?) {
+        migrateToMLSUserIDIn_MockMethod = method
+    }
+
+    func setMigrateToMLSUserIDIn_MockValue(_ value: MLSGroupID?) {
+        migrateToMLSUserIDIn_MockValue = value
+    }
```

Comment:
I didn't quite understand why you need these setters, could you please explain it to me?

Reply:
You could try to set the values without func, the compiler will complain ;)
In an actor it is forbidden to write to properties without async context, because then it would not be protected...

Reply:
But why do we need the setters? Wouldn't it be sufficient to set the mocks in an async test?